### ISSUE TITLE
Documentation: Change project title in the CLI's readme and doc to npm project name

### ISSUE THREAD

**Issue by Ifycode** [code-collabo/node-mongo-cli#161]
#### Task description
Change the main header (i.e. project title) in both the CLI's [readme](https://github.com/code-collabo/node-mongo-cli) and [documentation](https://code-collabo.gitbook.io/node-mongo/) to the [name of the project on npm](https://www.npmjs.com/package/@code-collabo/node-mongo-cli).

**Comment by izhar360** [comment]
@Ifycode  hey where exactly? PLEASE

**Comment by Ifycode** [comment]
Hi @izhar360 
If you mean the name of the project on npm, see screenshot below (I cropped/cut it out from the npm page link which I added in the task description above). That should be your new folder. If you are looking for where to change the doc from, check the doc folder in the root of this repo.

![Screenshot_20220303-130627](https://user-images.githubusercontent.com/45185388/156561872-310bc05d-4b0a-4842-b0eb-d087c28f8e7a.jpg)
